src/lustre/lustreAst.ml

```diff
@@ -637,69 +646,48 @@ and pp_print_field_assign ppf (i, e) =
 
 (* Pretty-print a Lustre type *)
 and pp_print_lustre_type ppf = function
-
+  | TVar (_, i) -> pp_print_ident ppf "i"
```

Comment:
Isn’t the identifier `i` relevant when printing the type?

Reply:
This is a type variable, so the identifier will store information like `'a` i.e. the name of the type variable.

Reply:
Why are you not printing the name of the type variable? If you only print the string literal "i", it is less informative. Isn’t it?

Reply:
Oh my. You are definitely right. I am blind. :D

Reply:
This is fixed now.